server/migrations/20170623075621-create-comment.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Comments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      gift_id: {
+        type: Sequelize.INTEGER,
+        unique: false,
+        allowNull: false,
+        references: {
+          model: 'Gifts',
+          key: 'id'
+        }
+      },
+      user_id: {
```

Comment:
As I understand, the guest is a user (Guest.id = User.id) and I'll get his values (email, first_name, last_name) from the table User and Profile, and with donor will be as well, they both have an opportunity to comment... I should think about this